snowshu/core/replica/replica_factory.py

```diff
@@ -43,8 +46,11 @@ def _execute(self,
             return "No relations found per provided replica configuration, exiting."
 
         # TODO replica container should not be started for analyze commands
+        target = target if target is not None else DOCKER_TARGET_CONTAINER
```

Comment:
Since `initialize_replica` expects the image name and not the container name, the default `target` shouldn't have a value since the base image is determined by the concrete target adapter.